app/assess/routes.py

```diff
@@ -235,9 +235,13 @@ def landing():
     ).assessment_deadline
 
     post_processed_overviews = (
-        get_assessment_progress(application_overviews)
-        if application_overviews
-        else []
+        (
+            get_assessment_progress(application_overviews)
+            if application_overviews
+            else []
+        )
+        if not Config.USE_LOCAL_DATA
```

Comment:
This is a bit hacky.  It's way for the tests to not call the post request since we don't have any `get_local_data` concept for post requests setup at the moment.

Reply:
Can we maybe add a TODO/ticket for this so it doesn't get forgotten?

Reply:
Yeah hmmm it does feel a bit dirty having this in the main code... Maybe as it is a temporary fix to enable tests to pass perhaps the workaround should go in the tests file instead (with a TODO to note it). Also have you added a ticket to do the full fix (if not time to do it now)?

Reply:
Added in https://github.com/communitiesuk/funding-service-design-assessment/pull/132/commits/bca54c2337d16c746cb7d2318dcfbfc54ff5458b, thanks